NSS-Day-Cohort-68/rare-api-rare-team-1-1
# Remove db.sqlite3 from github

Based on our discussion this morning, db.sqlite3 should not be stored on github. This pull request aims to remove db.sqlite3 from being tracked on github. Once this change is tested, approved, and merged into dev, each person on the team must pull the updated version of dev onto their local repository. Then, dev will be merged into main and everyone will have to pull the updated version of main as well. 

No files were changed.

Change made: In terminal while on this branch ("bug/remove-db-from-tracking"), the following command was run:
git rm --cached db.sqlite3.

To test, do the following:
* git fetch
* git checkout bug/remove-db-from-tracking
* make a change to db.sqlite3 on your local computer.
* git add --all
* git status
* verify that db.sqlite3 is not included in the staged files
* no need to commit the files. this test was designed to verify that db.sqlite3 is not being tracked while on this branch.
